### DISCUSSION TITLE
Resetting autosave and commit function in codespace

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#558] [Q&A] [answered]
I'm afraid I've messed up my codespace settings. I changed an autosave function because it seemed codespace was constantly updating and now I notice the changes are not being updated and I can't commit changes. If I go back to a previously saved point would that fix it or is there a better option?

**Comment by BioNCphilic** [comment]
OK - so I tried opening the species distribution assignment in visual code..... changes are not being tracked there either. This screenshot is after several edits were made.
![Screenshot 2024-10-15 000338](https://github.com/user-attachments/assets/978da8b8-db13-4372-9b4f-c93d925d5d09)

**Comment by eculler** [comment, marked as answer]
@BioNCphilic can you clarify what you mean by you 'can't commit changes'?

A potential quick fix would be to download your files manually, create a new Codespace, and then upload them again. Or, if your environment is set up you can work on your computer. However, if you have settings sync on, this might not work.

I believe that if you go to settings (from the gear in the lower left) you can also reset settings to the default? You could try that too.

**Reply by BioNCphilic** [reply]
Thanks - I'll give it a try. Should my codespace have the auto-delete checked before I open it? I think that's where I started messing with settings. I'll try to make office hours today (2pm MT or 4pm ET).
